contents/views.py

```diff
@@ -1,3 +1,103 @@
 from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.serializers import ModelSerializer
+from rest_framework.views import APIView
+
+from contents.models import Contents_table
+
 
 # Create your views here.
+
+class ContentsSerializer(ModelSerializer):
+    class Meta:
+        model = Contents_table
+        fields = ['title', 'detail']
+
+
+class ContentsCreateAPI(APIView):
+    def get(self, request):
+        contents = Contents_table.objects.title
```

Comment:
filter 쿼리나 all 쿼리가 필요해 보입니다.
```suggestion
        contents = Contents_table.objects.filter()
```